src/repository/models.py

```diff
@@ -842,9 +854,49 @@ class VersionQueue(models.Model):
     date_decision = models.DateTimeField(blank=True, null=True)
     approved = models.BooleanField(default=False)
 
+    title = models.CharField(
+        max_length=300,
+        help_text=_('Your article title'),
+    )
+    abstract = models.TextField(
+        blank=True,
+        null=True,
+        help_text=_(
+            'Please avoid pasting content from word processors as they can add '
+            'unwanted styling to the abstract. You can retype the abstract '
+            'here or copy and paste it into notepad/a plain text editor before '
+            'pasting here.',
+        )
+
+    )
+
     def approve(self):
         self.date_decision = timezone.now()
         self.approved = True
+
+        # Update the current version to have the Preprint's current title
+        # and abstract.
+        current_version = self.preprint.current_version
+        current_version.title = self.preprint.title
+        current_version.abstract = self.preprint.abstract
```

Comment:
Now that we established 'blank' means unchanged, should we handle it here and in the VersionForm as well? Otherwise, all future `VersionQueue` and `PreprintVersion` objects will still have those redundant titles/abstracts. If this is desirable, then I'd suggest still adding the migration to backpopulate the past records for avoidance of doubt.

Reply:
Based on discussion earlier:

- add a migration to update existing version queue objects
- make fields not null